spectator/registry.h

```diff
@@ -52,6 +52,11 @@ struct meter_map {
     return meters_.size();
   }
 
+  auto contains(const Id& id) -> bool {
```

Comment:
Don't think these are needed.

Reply:
Ah, right, the `contains` access to `meters_` does not require locking the mutex, while getting a `at` meter does.